assets/src/pages/room/VideochatSection.tsx

```diff
@@ -151,15 +151,18 @@ export const VideochatSection: FC<Props> = ({ peers, localPeer, showSimulcast, w
 
   const { pinnedTiles, unpinnedTiles } = takeOutPinnedTiles(allTilesConfig, pinningApi.pinnedTileIds);
   const isSomeTilePinned = pinnedTiles.length > 0;
+  const isSomeTileUnpinned = unpinnedTiles.length > 0;
```

Comment:
I'm not sure, but `some` here sounds off to me, I'd rather use any
```suggestion
  const isAnyTilePinned = pinnedTiles.length > 0;
  const isAnyTileUnpinned = unpinnedTiles.length > 0;
```